--- node_modules/@pedroslopez/moduleraid/moduleraid.js	2023-03-08 02:06:53.734834902 +0000
+++ moduleraid.js	2023-03-08 02:08:34.843553496 +0000
@@ -6,7 +6,7 @@
  * https://github.com/pedroslopez/moduleRaid/blob/master/LICENSE
  */
 
-const moduleRaid = function () {
+const mrd = function () {
   moduleRaid.mID  = Math.random().toString(36).substring(7);
   moduleRaid.mObj = {};
 
@@ -67,7 +67,7 @@
 }
 
 if (typeof module === 'object' && module.exports) {
-  module.exports = moduleRaid;
+  module.exports = mrd;
 } else {
-  window.mR = moduleRaid();
+  window.mR = mrd();
 }
